SaintCoinach/ex.json

```diff
@@ -10218,6 +10218,9 @@
       "sheet": "PublicContent",
       "defaultColumn": "Name",
       "definitions": [
+        {
+          "name": "Key"
+        },
```

Comment:
We might see confusing conflicts with another Key column in here.  Maybe change this to Type?